forge/forge/op/eval/forge/tm.py

```diff
@@ -1413,6 +1378,35 @@ def decompose(type, attr, dc, inputs):
                 rank -= 1
             dc.fuse(result)
             return
+def create_row_picker_matrix(row_indices,  lhs_num_cols, lhs_num_channels=None, lhs_batch_size=None):
+    """
+    Create a sparse matrix that picks rows from a matrix.
```

Comment:
Function docs seem out-of-date. Can you update it to batter match the intended purpose? 

Also, a minor example will be beneficial for easier future understanding :))